assets/dev/scss/editor/preview/preview.scss

```diff
@@ -23,6 +23,39 @@
 
 @import "../flex-icons.scss";
 
+
+// mixins
+@mixin e-grid-outline-base {
+	padding-top: var( --padding-top );
```

Comment:
I would remove all the empty spaces inside `var()'`.

Reply:
Correct, my mistake :)